README.md

```diff
@@ -128,10 +128,76 @@ function PotatoNotification({ notificationCode, t }) {
 export default withTranslators(PotatoNotification)
 ```
 
+## Via [`babel-plugin-react-gettext-parser`](http://github.com/alexanderwallin/babel-plugin-react-gettext-parser)
```

Comment:
Nice to have a preset config! Make it an `###` so that it falls under the *Usage*. 👍